### ISSUE TITLE
Allow tagging with a repository name after the build

### ISSUE THREAD

**Issue by deinspanjer** [dnephin/dobi#69]
In an image, you must give the repository name in the image: attribute, and any entries in the tags: attribute are only appended to the repository name as "version tags".

In non-dobi builds, we build with an initial simple short name:
`docker build -t myco/myapp .`
and then afterward, we tag it for the registry:
`docker tag myco/myapp 123532365.aws.com/ecr/myco/myapp:latest`

but I'm not able to do this in Dobi, I have to give the long tag name as the image name.

**Comment by cescoferraro** [comment]
@deinspanjer Is that what you expect?

image=mongo-image:
    image: myco/myapp
    tags: [latest]
    pull: "once"
    alternatives: ["123532365.aws.com/ecr/myco/myapp"]

**Comment by dnephin** [comment]
I wonder if we could do something like this:

```yaml
image=myapp:
  ...

alias=tag-myapp:
  tasks:
    - 'myapp:tag(123532365.aws.com/ecr/myco/myapp)'

```

Although tag already has a behaviour, which would mean it has two forms.

**Comment by deinspanjer** [comment]
Ideally, I'd just like the tags property list to take the same types of values as the `docker tag` command does: https://docs.docker.com/engine/reference/commandline/tag/

**Comment by cescoferraro** [comment]
@deinspanjer #70 solves your problem

**Comment by deinspanjer** [comment]
Yep, the actual tag labels cannot contain slashes in them, so testing for a / and using the entire string up to any : as a new image name tag would be correct parsing logic.

#70 should unblock me, so I'll try it out later this week. need to replace the work-around docker tag commands I'm currently using.

**Comment by dnephin** [comment]
Just merged #70 which will be in 0.9
